lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -234,6 +234,8 @@ previewers.new_buffer_previewer = function(opts)
         buf_delete(bufnr)
       end
     end
+    -- enable resuming picker with existing previewer to avoid lookup of deleted bufs
+    bufname_table = {}
```

Comment:
I can't find anywhere that this is being used (on first search, just in github browser)

Reply:
getting a new table ref here should be good. I think we can ignore that comment :laughing: